Makefile

```diff
@@ -1,46 +1,34 @@
-.PHONY: build clean
+.PHONY: build clean test release
 OUTPUT = riff
-OUTPUT_LINUX = $(OUTPUT)-linux
-OUTPUT_WINDOWS = $(OUTPUT)-windows
-BUILD_FLAGS =
-
-ifeq ($(OS),Windows_NT)
-    detected_OS := Windows
-else
-    detected_OS := $(shell sh -c 'uname -s 2>/dev/null || echo not')
-endif
-
-ifeq ($(detected_OS),Linux)
-	BUILD_FLAGS += -ldflags "-linkmode external -extldflags -static"
-endif
-
-
 GO_SOURCES = $(shell find cmd pkg -type f -name '*.go')
 
 build: $(OUTPUT)
 
-vendor: Gopkg.toml
-	dep ensure
+vendor: Gopkg.lock
+	dep ensure -vendor-only
+
+Gopkg.lock: Gopkg.toml
+	dep ensure -update
 
 test: build
 	go test -v ./...
 
 $(OUTPUT): $(GO_SOURCES) vendor
-	go build github.com/projectriff/riff-cli/cmd/riff
+	go build -o $(OUTPUT) cmd/riff/main.go
+
+release: $(GO_SOURCES) vendor
+	GOOS=darwin   GOARCH=amd64 go build -o $(OUTPUT) cmd/riff/main.go && zip -m riff-darwin-amd64.zip  $(OUTPUT)
```

Comment:
would prefer tar -czf to create a .tgz archive - seems more appropriate for Linux and also macOS - can do on merge

Reply:
Thought about that, but then realized may not be easy for windows.
zip for win, and tgz for Mac/Linux is certainly the best option